x-pack/test/api_integration/apis/ml/modules/recognize_module.ts

```diff
@@ -138,7 +139,12 @@ export default ({ getService }: FtrProviderContext) => {
       user: USER.ML_POWERUSER,
       expected: {
         responseCode: 200,
-        moduleIds: ['auditbeat_process_hosts_ecs', 'security_linux', 'siem_auditbeat'],
+        moduleIds: [
+          'auditbeat_process_hosts_ecs',
+          'security_auth',
```

Comment:
@peteharverson I noticed that in the testing output, but `security_auth` has been added to the auditbeat testset  and yet that test still fails.

Reply:
@ajosh0504 From what I can see in the test log it looks like we're now expecting the `security_auth` module to be recognized in the `ft_module_auditbeat` index pattern, but it is not. If we think that it's ok it's not recognized there, we can remove it from the expected modules for this dataset.

Reply:
@pheyos Any idea it might not be recognizing the module?

Reply:
@ajosh0504 I'd need to double check the dataset. But the new module checks for `"event.category": "authentication"`. And it it is not recognized in an index pattern, it means that the documents don't have this field or have a different value for this field. I'll take a closer look and report back here.

Reply:
@ajosh0504 I've checked the `ft_module_auditbeat` dataset and it has `"event.category": "audit-rule"` (see screenshot), so it's ok that it doesn't match.
![image](https://user-images.githubusercontent.com/1945390/121542043-16713880-ca08-11eb-8e8a-f8547892fa98.png)

Reply:
Yes event.category was added last year and this Auditbeat data may be from 2019 so that test will have to be skipped until we have newer data.

Reply:
I've created a reminder issue to decide whether or not to update the dataset, see #101910.
For now it's fine to not have the `security_auth` in the list of expected modules for the "old" auditbeat dataset.